teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>This methods prevents data leaking &gt; This methods <strong>prevent</strong> data leaking</p><p>One (or several, LSCCO) clusters &gt; &nbsp;One (or several, LSCCO) <strong>cluster(s)</strong></p><p>those released until certain date &gt; &nbsp;those released until <strong>a </strong>certain date</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>